### ISSUE TITLE
No buffer zone between script editor left panel edge and start of script text

### ISSUE THREAD

**Issue by josephj11** [autokey/autokey#668]
### Has this issue already been reported?

- [X] I have searched through the existing issues.

### Is this a question rather than an issue?

- [X] This is not a question.

### What type of issue is this?

UI/Usability

### Which Linux distribution did you use?

kubuntu 18.04

### Which AutoKey GUI did you use?

GTK

### Which AutoKey version did you use?

0.96.0 Beta 10

### How did you install AutoKey?

From GitHub debs

### Can you briefly describe the issue?

It appears to be impossible to select script text from the beginning of a line using the mouse in the script editor. It either selects the second character or tries to resize the panel to the left.

There are awkward workarounds.

### Can the issue be reproduced?

Always

### What are the steps to reproduce the issue?

Select a script
Try to select text in the script from the beginning of a line in the script editor panel.

### What should have happened?

It should be easy to place the mouse cursor to start selecting from the start of the line.

### What actually happened?

I can't get this to work at all. I had to select right to left to select the beginning of the line at all.

### Anything else?

This works fine in autokey-qt where there is a large left margin between the panel edge and the start of text.

**Comment by Elliria** [comment]
Yes. I'll add a big, huge plus one to this (thumbs-up was the closest icon to a +1). It's such an annoyance. You go to select the text in the editor and you adjust the width of the left and right panes instead [sigh]. It's the same for me in that I either have to select from right to left (which is totally unnatural) or left-click somewhere in the right pane and then switch to the keyboard to do the selecting, and either of those happen only after already having accidentally adjusted the pane widths [sigh].

**Comment by josephj11** [comment]
@Elliria Learn Hebrew LOL. (It's written right to left.)

**Comment by Elliria** [comment]
Our luck, if AutoKey were being run on a Hebrew machine, it would have the opposite problem and the right margin would have the issue.
